data.sql

```diff
@@ -217,4 +217,168 @@ WHERE animals.name in ('Charmander', 'Squirtle', 'Blossom') and owners.full_name
 UPDATE animals 
 	SET owner_id = owners.id
 FROM owners 
-WHERE animals.name in ('Angemon', 'Boarmon') and owners.full_name = 'Dean Winchester';
\ No newline at end of file
+WHERE animals.name in ('Angemon', 'Boarmon') and owners.full_name = 'Dean Winchester';
+
+
+/*Populate vets table*/
+
+INSERT INTO vets (
+	name,
+	age,
+	date_of_graduation
+) VALUES (
+	'William Tatcher', 
+	45,
+	'2000-04-23'
+);
+
+INSERT INTO vets (
+	name,
+	age,
+	date_of_graduation
+) VALUES (
+	'Maisy Smith', 
+	26,
+	'2019-01-17'
+);
+
+INSERT INTO vets (
+	name,
+	age,
+	date_of_graduation
+) VALUES (
+	'Stephanie Mendez', 
+	64,
+	'1981-05-04'
+);
+
+INSERT INTO vets (
+	name,
+	age,
+	date_of_graduation
+) VALUES (
+	'Jack Harkness', 
+	38,
+	'2008-06-08'
+);
+
+
+/*Populate specializations table*/
+
+INSERT INTO specializations (specie_id, vet_id) 
+SELECT species.id , vets.id
+FROM species
+INNER JOIN vets ON vets.name = 'William Tatcher' AND species.name = 'Pokemon';
+
+INSERT INTO specializations (specie_id, vet_id) 
+SELECT species.id , vets.id
+FROM species
+INNER JOIN vets ON vets.name = 'Stephanie Mendez' AND species.name = 'Pokemon';

```

Comment:
- [ ] Good job so far 👍 Stephanie Mendez specializes both in Digimon and Pokemon, therefore you are to also add this information into the data.sql. Right now you have only placed Stephanie with the Pokemon. Kindly add the code to associate her also with Digimon 👍